### DISCUSSION TITLE
Welcome! Please introduce yourself. 😊

### DISCUSSION THREAD

**Discussion by deborahtrez** [curiouslearning/CuriousReaderInteractiveFunctionsH5P#225] [#OSEQ Hackathon]
Hi @everyone 🙋🏽‍♀️.
I just wanted to say hi and welcome you to the Curious Reader community🎉—I'm Deborah and it's a joy to have you here.
I will be your main facilitator for the Curious Reader hackathon. 😎
Please don't be shy to say _hello_ and introduce yourself! 😌 Infact, we encourage it. 
I hope you will have a great time creating and submitting your contributions and I want *you to know that if you have any questions, you're welcome to ask me [@deborahtrez ] or the community anything!

**Comment by Samsonroyal** [comment]
Hello I am Samson and I’m glad to be here. 😊

**Reply by burrage** [reply]
Hey @Samsonroyal ! Thanks for saying hi and I'm looking forward to seeing what you get up to during the session.

**Reply by deborahtrez** [reply]
Hi Samson, How are you? It's good to hear from you. 😊

**Reply by Samsonroyal** [reply]
I am looking forward to building.. can't wait

**Comment by DoozyDoz** [comment]
Hello I am Muhammad, nice to meet y'all

**Reply by deborahtrez** [reply]
Hi Doozy, how are you?! Its is nice to meet you too. I hope you are not having any difficulties with your task?

**Comment by lailahgrant** [comment]
Hi, I'm Lailah. Happy to be here.

**Reply by deborahtrez** [reply]
Hi Lailah!

**Comment by fxinyangat** [comment]
Hi everyone, my name is Xavier, glad to be here

**Reply by deborahtrez** [reply]
Hi Xavier. Its good to hear from you!.😊

**Comment by Mik-256** [comment]
**Hello **everyone
I believe we are all well, Just came online, I hope am not late for the party
Tell me what did I miss?****

**Reply by deborahtrez** [reply]
Hi Mik, I dont know if you missed much but a lot of people have been working tirelessly on their tasks. What are you working on?

**Reply by Mik-256** [reply]
Hello, Debbie, I wanted to know how much time I have left because Iseem to have checked my inbox a bit late late. Am working solo on 5 books. authoring, translating and compiling

**Reply by deborahtrez** [reply]
Hi Mik-256

The contributhon ends on 14th August. But we’ll probably close submissions early on so that the Curious Learning team can have time to review all the submissions and pick out winners. That might be 12th or 13th August.

If you’re working solo, please don’t feel pressured to work on a lot of tasks alone. Feel free to get help from any one of our participants in the emailing list. And in my experience, it helps to work slowly but consistently so that you don’t get overwhelmed. 😊
 
Happy Authoring!

**Comment by MarcDemo** [comment]
I everyone am Mark Demo glad to be part of this, excited to take part in more coming activities.

**Comment by Bennahk** [comment]
> Hi @everyone 🙋🏽‍♀️. I just wanted to say hi and welcome you to the Curious Reader community🎉—I'm Deborah and it's a joy to have you here. I will be your main facilitator for the Curious Reader hackathon. 😎 Please don't be shy to say _hello_ and introduce yourself! 😌 Infact, we encourage it. I hope you will have a great time creating and submitting your contributions and I want *you to know that if you have any questions, you're welcome to ask me [@deborahtrez ] or the community anything!

@deborahtrez , thanks for the mentorship during the OSEQ event. With your guidance, I was able to achieve so much in little time

**Reply by deborahtrez** [reply]
Aaaww! Thank you!🥹❤️ 
I am so happy that you find my mentorship beneficial.
Thank you for participating in this track of the hackathon. ☺️

**Comment by Bennahk** [comment]
You are welcome
